snuba/util.py

```diff
@@ -414,7 +414,7 @@ def raw_query(body, sql, client, timer, stats=None):
     project_id = project_ids[0] if project_ids else 0  # TODO rate limit on every project in the list?
     stats = stats or {}
     grl, gcl, prl, pcl, use_cache, uc_max = state.get_configs([
-        ('global_per_second_limit', 1000),
+        ('global_per_second_limit', None),
```

Comment:
This is the main change. Defaulting this to None while leaving our options open to re-enable global per-second limiting.

Need to delete the `global_per_second_limit` key in the config for this to start working